### DISCUSSION TITLE
Visualize post_transforms

### DISCUSSION THREAD

**Discussion by Gijz33** [Project-MONAI/tutorials#403] [Q&A]
Hi,

I have a question regarding visualizing post transformations using SupervisedTrainer and SupervisedEvaluator.
How can I 'reach' and visualize the output of postprocessing in SupervisedEvaluator (val_post transform). 
I assume the solution for SupervisdTrainer will be the same?

**Comment by Nic-Ma** [comment]
Hi @Gijz33 ,

Thanks for your interest here.
I think maybe you can use the `TensorBoardImageHandler` to visualize the predictions:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/tensorboard_handlers.py#L232

Thanks.

**Reply by Gijz33** [reply]
Thank you for your help @Nic-Ma!
